doc/DeveloperGuide.md

```diff
@@ -55,10 +55,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 
 Priority | As a ... | I want to ... | So that I can...
 -------- | :-------- | :--------- | :-----------
+`* * *` | new user | import my address book | continue using it
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | personalize my contacts | find them more easily
```

Comment:
Usually, ui should be given a lower priority as functionality is more important.
If you want to give ui a high priority, you will need to focus your ui as the selling point.
Also, `find them more easily` should be more descriptive e.g. in terms of? visually? etc.